docs/src/modules/components/BaseUIComponents.js

```diff
@@ -4,136 +4,136 @@ import CardMedia from '@mui/material/CardMedia';
+      href: '/base-ui/react-select/',
     },
     {
       title: 'Slider',
-      srcLight: '/static/images/base-ui-components/slider-light.png',
-      srcDark: '/static/images/base-ui-components/slider-dark.png',
-      href: '/base/react-slider/',
+      srcLight: '/static/base-ui/react-components/slider-light.png',
+      srcDark: '/static/base-ui/react-components/slider-dark.png',
+      href: '/base-ui/react-slider/',
     },
     {
       title: 'Snackbar',
-      srcLight: '/static/images/base-ui-components/snackbar-light.png',
-      srcDark: '/static/images/base-ui-components/snackbar-dark.png',
-      href: '/base/react-snackbar/',
+      srcLight: '/static/base-ui/react-components/snackbar-light.png',
+      srcDark: '/static/base-ui/react-components/snackbar-dark.png',
+      href: '/base-ui/react-snackbar/',
     },
     {
       title: 'Switch',
-      srcLight: '/static/images/base-ui-components/switch-light.png',
-      srcDark: '/static/images/base-ui-components/switch-dark.png',
-      href: '/base/react-switch/',
+      srcLight: '/static/base-ui/react-components/switch-light.png',
+      srcDark: '/static/base-ui/react-components/switch-dark.png',
+      href: '/base-ui/react-switch/',
     },
     {
       title: 'Table Pagination',
-      srcLight: '/static/images/base-ui-components/pagination-light.png',
-      srcDark: '/static/images/base-ui-components/pagination-dark.png',
-      href: '/base/react-table-pagination/',
+      srcLight: '/static/base-ui/react-components/pagination-light.png',
+      srcDark: '/static/base-ui/react-components/pagination-dark.png',
+      href: '/base-ui/react-table-pagination/',
     },
     {
       title: 'Tabs',
-      srcLight: '/static/images/base-ui-components/tabs-light.png',
-      srcDark: '/static/images/base-ui-components/tabs-dark.png',
-      href: '/base/react-tabs/',
+      srcLight: '/static/base-ui/react-components/tabs-light.png',
+      srcDark: '/static/base-ui/react-components/tabs-dark.png',
+      href: '/base-ui/react-tabs/',
     },
     {
       title: 'Textarea Autosize',
-      srcLight: '/static/images/base-ui-components/textarea-light.png',
-      srcDark: '/static/images/base-ui-components/textarea-dark.png',
-      href: '/base/react-textarea-autosize/',
+      srcLight: '/static/base-ui/react-components/textarea-light.png',
+      srcDark: '/static/base-ui/react-components/textarea-dark.png',
+      href: '/base-ui/react-textarea-autosize/',
     },
   ];
 }
 
-function BaseUIComponents() {
+export default function BaseUIComponents() {
   return (
     <Grid container spacing={2} sx={{ pt: 2, pb: 4 }}>
       {components().map((component) => (
         <Grid item xs={12} sm={4} sx={{ flexGrow: 1 }} key={component.title}>
           <Card
-            component="a"
+            component={Link}
+            noLinkStyle
             variant="outlined"
-            rel="nofollow"
-            target="_blank"
```

Comment:
I believe opening in a new tab is a strange UX, none of the benchmark in https://www.notion.so/mui-org/Build-a-components-index-page-20e993e49ee9473fac1ef0aa1a00ab39 do this.

Reply:
Makes sense, good catch!